code/modules/client/preferences_savefile.dm

```diff
@@ -252,7 +252,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	//Species
 	var/species_id
 	S["species"]			>> species_id
-	var/list/roundstart_races = CONFIG_GET(keyed_flag_list/roundstart_races)
+	var/list/roundstart_races = GLOB.roundstart_races.Copy()
```

Comment:
why copying the list?

Reply:
I don't remember actually!

Reply:
no reason for it

Reply:
I know, I think I copied it from the area where I actually needed Copy for some reason